src/classes/CON_DeleteContactOverride_CTRL.cls

```diff
@@ -63,8 +66,11 @@ public with sharing class CON_DeleteContactOverride_CTRL {
     * account should be deleted.
     */ 
     public CON_DeleteContactOverride_CTRL(ApexPages.StandardController stdController) {
-        retURL = ApexPages.currentPage().getParameters().get('returl');
-        
+        sforceUser = UserInfo.getUiThemeDisplayed().contains('Theme4');
```

Comment:
Yes it is here: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_userinfo.htm